addons/packages/kapp-controller/0.30.0/bundle/config/schema.yaml

```diff
@@ -18,10 +18,10 @@ kappController:
     #@schema/desc "The coreDNSIP will be injected into /etc/resolv.conf of kapp-controller pod"
     #@schema/nullable
     coreDNSIP: ""
-    #@schema/desc "HostNetwork of kapp-controller deployment"
+    #@schema/desc "Host networking requested for kapp-controller pod. Use the host's network namespace. If this option is set, the ports that will be used must be specified"
```

Comment:
thought: How about just the first sentence "Host networking requested for kapp-controller pod" and omitting the other sentences after that, as we don't have any instructions how to set ports

Reply:
Fixed.